lib/models/graph-object.js

```diff
@@ -56,12 +56,6 @@ function GraphModelFactory (
                 var obj = this.toObject();
                 return Constants.Task.ActiveStates.indexOf(obj._status) > -1;
             }
-        },
-        $indexes: [
-            {
-                keys: { instanceId: 1 },
```

Comment:
@yyscamper that's why I remove some indexes, they're duplicated, and I could check the indexes by db.xx.getIndexes() to see if the database index are affected instead of scaling test.

actually, not only 'unique: true' would affect the index, but also 'primaryKey: true' so I prefer attributes keep the same with before, and check if new added index is duplicated, if yes, remove them, if not, leave there, to reduce the impacts.